plugins/outputs/signalfx/signalfx.go

```diff
@@ -55,6 +60,9 @@ func NewSignalFx() *SignalFx {
 		EventIngestURL:     "https://ingest.signalfx.com/v2/event",
 		Exclude:            []string{""},
 		Include:            []string{""},
+		dps:                make(chan *datapoint.Datapoint, 100000),
```

Comment:
should we makw channel size configurable?

Reply:
How? your New doesn't take any params... could offer a resize method...